File: PHP/uberVan.php
<?php
require_once("car.php");

class UberVan extends Car{
    public $typeCarAccepted;
    public $material;

    public function __construct($license, $driver, $typeCarAccepted, $material){
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->material = $material;
    }
    public function setPassengers($passengers) {
    
        if ($passengers == 6) {
            $this->passenger = $passengers;
        }
        else {
            echo "Necesitas asignar 6 pasajeros 
    ";
        }
    
    }
}

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car{

    Map<String,Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String licence, Account driver){//, Map<String,Map<String,Integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
        super(licence, driver);
        //this.typeCarAccepted = typeCarAccepted;
        //this.seatsMaterial = seatsMaterial;
    }

    @Override
    public void setPassenger(Integer passengers) {
        //super.setPassenger(passengers);
        if(passengers == 6){
            this.passengers = passengers;
        }else{
            System.out.println("<---------UberVan necesita un minimo de 6 pasajeros--------->");
        }
    }
}
